ibis-server/README.md

```diff
@@ -61,32 +61,33 @@ vim .env
 ```
 Install the dependencies
 ```bash
-make install
+just install
 ```
 Run the server
 ```bash
-make run
+just run
 ```
 
 ## Developer Guide
 
 ### Environment Setup
 - Python 3.11
-- Install `poetry` with version 1.7.1: `curl -sSL https://install.python-poetry.org | python3 - --version 1.7.1`
-- Install pre-commit hooks: `make pre-commit-install`
-- Execute `make install` to install the dependencies
-- Execute `make test` to run the tests
+- Install [poetry](https://github.com/python-poetry/poetry) with version 1.7.1: `curl -sSL https://install.python-poetry.org | python3 - --version 1.7.1`
+- Install [casey/just](https://github.com/casey/just)
+- Install [pre-commit](https://pre-commit.com) hooks: `just pre-commit-install`
+- Execute `just install` to install the dependencies
+- Execute `just test` to run the tests
```

Comment:
Moving to `just` is a good improvement. However, there're many requirement tools before starting to develop this project. How about implementing a `Makefile` to install all of them?

Reply:
As discussed offline, we leave this issue for the user to decide how to install the requirements themselves.